src/data/teams.json

```diff
@@ -1,6 +1,15 @@
 [
   {
     "title": ".NET Core",
+    "description": ".NET Core",
+    "platforms": [
+      {"name": "github", "label": "thenewboston", "link": "https://github.com/thenewboston-developers"},
```

Comment:
just a suggestion, not sure if needed.

but would it be better if we structure the platforms as something like below, since each team may be in charge of several repositories:

```
platforms: [
   { 
         name: "github", 
         links: [
             { label: "Website", url: "https://github.com/thenewboston-developers/Website"},
             { label: "Account Manager", url: "https://github.com/thenewboston-developers/Account-Manager"}
         ]
    }, ...
]
```

Reply:
Yeah, that's one way to do it. But my plan was to have it like this:

```
"platforms": [
      {"name": "github", "label": "thenewboston", "link": "https://github.com/thenewboston-developers/Website"},
      {"name": "github", "label": "thenewboston", "link": "https://github.com/thenewboston-developers/Account-Manager"},
      {"name": "slack", "label": "front-end", "link": "https://slack.com"},
 ]
```

So, instead of comma-separated values, we can display them on the next line with their own Github icon. Looks cleaner IMO.

Reply:
Ahhh i see. Yep sounds good!